drake/multibody/rigid_body_plant/visualization/show_time.py

```diff
@@ -0,0 +1,26 @@
+# Note that this script runs in the main context of drake-visulizer,
+# where many modules and variables already exist in the global scope.
+from director import lcmUtils
+import bot_core as lcmbotcore
+import time
+
+def onViewerDraw(msg, last_time = [0, 0]):
```

Comment:
`last_time` is an abuse of a default arg.  use a global variable, or even better, make this a class with a member variable